jenkins-schema.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -xe
 export DISPLAY=:99
-export GOVUK_APP_DOMAIN=test.alphagov.co.uk
-export GOVUK_ASSET_ROOT=http://static.test.alphagov.co.uk
+export GOVUK_APP_DOMAIN=test.alphagov.gov.uk
```

Comment:
That would imply it's a real domain. The usage of `test.gov.uk` is more analogous to our use of `dev.gov.uk`.